src/graphorm/index.ts

```diff
@@ -74,6 +74,13 @@ const obj = new GraphORM({
   CommentUpvote: {
     requiredColumns: ['createdAt'],
   },
+  UserStreak: {
+    fields: {
+      max: { select: 'maxStreak' },
+      total: { select: 'totalStreak' },
+      current: { select: 'currentStreak' },
```

Comment:
Why do we have separate names for the db columns and the graphql ones? Why don't we keep the same name?

Reply:
Took it as how it was desired from the DR. But we can probably keep the same names to reduce complexity. cc @DragosIorgulescu wdyt?

Reply:
Yeah you query `userStreak.something` right?
Would make sense to me to keep them inline.

Reply:
we can change the naming here to be the same, up to you if you want to use single word variations @sshanzel

Reply:
Alright. Let's do single-word names since we also had that on our RFC too 🥳